.changeset/silly-sheep-happen.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/theme-check-common': major
```

Comment:
Personally, I believe we could update this to be `minor`:

```suggestion
'@shopify/theme-check-common': minor
```

Otherwise, we will need to bump Theme Check to Theme Check 3, and I don't think we want to do that. What do you think @MateusJabour @albchu?